tutorials/text/annotations.py

```diff
@@ -59,6 +59,15 @@
 # For physical coordinate systems (points or pixels) the origin is the
 # bottom-left of the figure or axes.
 #
+# The following strings are also valid arguments for *textcoords*
+#
+# ==================  ========================================================
+# argument            coordinate system
+# ==================  ========================================================
+# 'offset points'     offset (in points) from the xy value
+# 'offset pixels'     offset (in pixels) from the xy value
```

Comment:
```suggestion
# 'offset points'     offset (in points = 1/72 inch) from the xy value
# 'offset pixels'     offset (in pixels) from the xy value
```

Reply:
Maybe sentence underneath the table linking out to the physical units tutorial?